### PR TITLE
PR for Lesson 05 Mailroom Part 3: mathewcm (also Lesson 7 commits piled up on this, so leaving this open)

### PR DESCRIPTION
**Pull request by mathewcmartin**

### CODE REVIEW

**Comment by mathewcmartin** [comment]
The actual lesson05 mailroom.py file is in ./student/mathewcm/lesson06 just to clarify!

File: students/mathewcm/lesson06/mailroom.py [hunk 1]
```diff
@@ -28,15 +28,22 @@ def return_to_menu():
...
+        donors_report.append((n,
+                      sum(don),
+                      len(don),
+                      (total[-1] / number[-1]),
+                      ))
+#        name.append(n)
+#        total.append(sum(don))
+#        number.append(len(don))
+#        avg.append(total[-1] / number[-1])
+    donors_report = sort(key=lambda y: y[1], reverse=True)
```
Comment (hosungs) [inline students/mathewcm/lesson06/mailroom.py]: Cool lambda!

**Comment by hosungs** [comment]
I see that this PR already absorbed your commits for Lesson 7, so I think we should leave this open, but enter your grades for the previous lessons (5 & 6). Just FYI...

**Comment by mathewcmartin** [comment]
Not sure what to do with this PR? Please advise.

**Comment by hosungs** [comment]
Sorry... I'll definitely review all your commits in this PR this evening and update your grades on Canvas, and then close the PR. In the meantime, hopefully no new commits are made, in which case we may not be able to close this PR again...

**Comment by mathewcmartin** [comment]
No more commits tonight. Sorry about that!

File: students/mathewcm/lesson07/html_render.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+class Element(object):
+
+    def __init__(self, content=None):
+        pass
+
+    def append(self, new_content):
+        pass
+
+    def render(self, out_file):
+        out_file.write("just something as a place holder...")
```
Comment (hosungs) [inline students/mathewcm/lesson07/html_render.py]: @mathewcmartin -- I've been waiting for your completed html_render.py with all the required classes (or at least most of them) implemented, but I don't see those other classes. I can't give a complete mark on this submission. Please do work on this assignment and complete it. Please do ask specific questions in class or at office hours.

**Comment by hosungs** [comment]
Our comments crossed. It's me who's sorry. I should have left comments and merged anyway, but then it becomes really hard to keep track of progress across multiple PRs, so I was waiting for your PR to be completed with more related commits, but apparently that wasn't communicated properly. I think it's better to merge anyway and wait for next PRs, so that's what I did. Sorry about all the confusion...

**Pull request merged**